query-service-impl/src/main/java/org/hypertrace/core/query/service/projection/ProjectionTransformation.java

```diff
@@ -422,4 +405,19 @@ private Filter updateFilterForComplexAttributeExpressionFromFilter(Filter origin
       return originalFilter;
     }
   }
+
+  private List<Filter> createFilterForComplexAttributeExpressionFromOrderBy(
+      List<OrderByExpression> orderByExpressionList) {
+    return orderByExpressionList.stream()
+        .map(OrderByExpression::getExpression)
+        .filter(this::isAttributeExpressionWithSubpath)
+        .map(Expression::getAttributeExpression)
+        .map(QueryRequestUtil::createContainsKeyFilter)
+        .collect(Collectors.toList());
+  }
+
+  private boolean isAttributeExpressionWithSubpath(Expression expression) {
```

Comment:
you already have defined this util elsewhere

Reply:
moved here because was not being used anywhere else.

Reply:
It was and should be, that's a bug. This is the condition we need to make the `mapValue` translation. Ref https://github.com/hypertrace/query-service/pull/107/files#r766680712

Reply:
Cannot redirect to the link.

Reply:
I'm guessing the comment got resolved or something, but it's the same line I keep commenting on https://github.com/hypertrace/query-service/pull/107/files#diff-c2c3a8f7fb70484b3563644ea25dde0c24675eabc2e010c8c0580a504684085eR272